src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -0,0 +1,289 @@
+     * by the `@client` query so that we can act on it if we encounter an ENS
+     * error.
+     *
+     * Based on that error we can determine if the colony is registered or not.
+     */
+    variables: dataVariables,
+  } = useColonyFromNameQuery({
+    variables: { address: '', name: colonyName },
+  });
+
+  const reverseENSAddress = dataVariables?.address;
+
+  const [
+    fetchTransction,
+    {
+      data: transactionData,
+      loading: transactionDataLoading,
+      error: transactionDataError,
+    },
+  ] = useTransactionLazyQuery();
+
+  const [
+    fetchUser,
+    { data: userData, loading: userDataLoading },
+  ] = useUserLazyQuery();
+
+  useEffect(() => {
+    if (
+      transactionHash &&
+      isTransactionFormat(transactionHash) &&
+      colonyData?.colony
+    ) {
+      fetchTransction({
+        variables: {
+          transactionHash,
+          colonyAddress: colonyData.colony.colonyAddress,
+        },
+      });
+    }
+  }, [fetchTransction, transactionHash, colonyData]);
+
+  useEffect(() => {
+    if (transactionData?.transaction?.from) {
+      fetchUser({
+        variables: { address: transactionData.transaction.from },
+      });
+    }
+  }, [fetchUser, transactionData]);
+
+  const fallbackUserData = useUser(transactionData?.transaction?.from || '');
+
+  const { Decorate } = new TextDecorator({
+    username: (usernameWithAtSign) => (
+      <UserMention username={usernameWithAtSign.slice(1)} />
+    ),
+  });
+
+  const titleDynamicValues = useMemo(
+    () => ({
+      user: (() => {
+        /*
+         * @NOTE Using a fallback profile allows us to display a user's address
+         * if he doesn't have a colony profile yet
+         */
+        const {
+          profile: { username, walletAddress },
+        } = userData?.user || fallbackUserData;
+
+        if (username && walletAddress) {
+          return <Decorate key={walletAddress}>{`@${username}`}</Decorate>;
+        }
+        if (walletAddress) {
+          return (
+            /*
+             * @NOTE This might not exist in the final title copy in this format
+             * but most likely we'll have "some" iteration of this for which we'll
+             * use this as base
+             */
+            <span className={styles.addressInTitle}>
+              <CopyableAddress>{walletAddress}</CopyableAddress>
+            </span>
+          );
+        }
+        return false;
+      })(),
+      name: (() => {
+        if (transactionData?.transaction?.events?.length) {
+          const { events } = transactionData.transaction;
+          /*
+           * Display the first event as the page title
+           * We might need to change this in the future
+           */
+          return events[0].name;
+        }
+        return false;
+      })(),
+    }),
+    [transactionData, fallbackUserData, userData],
+  );
+
+  if (!isTransactionFormat(transactionHash) || transactionDataError) {
```

Comment:
sooo... if transactionDataError then render?

Reply:
This is:
- if the hash is not the correct format
- Or there was an error fetching the local resolver (this could happen if we can't get either the transaction or the transaction receipt)